protos/sources/data-node/api/v2/trading_data.proto

```diff
@@ -1973,6 +1981,22 @@ message MarketConnection {
   PageInfo page_info = 2;
 }
 
+// Request for listing successor markets.
+message ListSuccessorMarketsRequest {
+  // Market ID that is a member of the succession chain. This can be the original market
+  // or any subsequent child market that succeeded it.
+  string market_id = 1 [(google.api.field_behavior) = REQUIRED];
+  // Flag to indicate whether or not to include the full succession chain, or only list
+  // the children of the given market ID. If true, the full succession chain is in included.
+  bool full_history = 2;
```

Comment:
In other APIs where we have booleans we name them like `include_settled`, `exclude_liquidity` and `live_only` (which should probably have been called something like `exclude_inactive`). I wonder if we should try to keep to something similar here too and call it `include_parents`?

@candida-d any opinions?

Reply:
I'd always vote for consistency. And now that you mention it, "full history" isn't the most obvious name for this boolean option. 

Would it be `include parent`, as presumably there's only one parent?

Reply:
Well the API will return the market requested and its children only if full_history is false, and if true, then it will return the full successor chain, i.e. the full history. I'm happy to change the name, but I don't think include_parents is quite accurate either.

Reply:
I suppose include_full_history would be more in line with what @wwestgarth is suggesting?